src/components/Table/Table.jsx

```diff
@@ -152,6 +155,26 @@ const Tr = React.createClass(createLucidComponentDefinition({
 	}
 }));
 
+const ResizableThContent = ({ children, onDrag, onDragEnd, onDragStart }) => (
```

Comment:
No, it's not. Our original render used it a couple of times, but we ended up being able to refactor the logic to only utilize this one time. We missed the opportunity to eliminate the component in all of our euphoria. ;)